source/types/Case.ts

```diff
@@ -7,6 +7,7 @@ export enum ApplicationStatusType {
   ACTIVE_SIGNATURE_COMPLETED = "active:signature:completed",
   ACTIVE_SIGNATURE_PENDING = "active:signature:pending",
   ACTIVE_SUBMITTED = "active:submitted",
+  ACTIVE_COMPLETION_SUBMITTED = "active:completion:submitted",
```

Comment:
We need to add `ACTIVE_RANDOM_CHECK_SUBMITTED` as well. This is the good old 'Stickprovskontroll'
`active:randomCheck:submitted`

Reply:
Created separate task for that one:
https://app.clickup.com/t/1vzckax